src/Filters/filt.jl

```diff
@@ -561,6 +561,7 @@ function _fftfilt!(out::AbstractArray{T}, b::AbstractVector{T}, x::AbstractArray
     filterft = similar(tmp2)
     copyto!(tmp1, b)
     tmp1[nb+1:end] .= zero(T)
+    broadcast!(*, tmp1, tmp1, normfactor)
```

Comment:
Couldn't this be combined with the `copyto!` above to `tmp1[1:nb] .= b .* normfactor`?

Reply:
Just made that change and pushed to my fork, though for some reason it's not showing up here.